org.flathub.flatpak-external-data-checker.metainfo.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="console-application">
+  <id>org.flathub.flatpak-external-data-checker</id>
+  <name>Flatpak External Data Checker</name>
+  <summary>A tool for checking if the external data used in Flatpak manifests is still up to date</summary>
+  <url type="homepage">https://github.com/flathub/flatpak-external-data-checker</url>
```

Comment:
I would submit this upstream if possible.

Reply:
Opened flathub/flatpak-external-data-checker#67